Gemfile

```diff
@@ -71,6 +71,9 @@ gem "pg", group: :pgsql, require: false
 # Bit fields for ActiveRecord (https://github.com/pboling/flag_shih_tzu)
 gem "flag_shih_tzu" #, "~> 0.3.23"
 
+# Required for the `import` statements at the top of the Stat generators
+gem "activerecord-import"
```

Comment:
I think this gem is for the bulk-import of data and generating minimal SQL statements rather than for the Ruby-import keyword.

I've reverted the import statements from the services, but the commit that added this references `imagemagick` so I wanted to double check before removing this

Reply:
ah yes. I thought I had reverted that change.